src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -19,18 +19,49 @@ If you're new to the topic of nodes, we recommend first checking out our user-fr
 
 ## What are nodes and clients? {#what-are-nodes-and-clients}
 
-A "node" is a computer running Ethereum client software. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. Up until The Merge, a single piece of software is required to run a full node (or two to run a mining node). After The Merge, two pieces of client software are required to run a full node (three to run a validator node), one to handle and gossip transactions (execution client), one to handle block gossip and fork choice (consensus client) and an optional validator client that handles block production and "voting" on blocks received from its peers.
+"Node" refers to a running piece of client software. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate.
 
-You can see a real-time view of the Ethereum network by looking at this [map of nodes](https://etherscan.io/nodetracker).
+Post-Merge Ethereum consists of two networks: the execution layer and the consensus layer. Both networks are run by different client software. On this page, we'll refer to them as the execution client and consensus client. 
```

Comment:
> Post-Merge Ethereum consists of two networks

_Should_ we distinguish it as "two networks"? Makes things a bit confusing IMO to say that Ethereum will now be "two networks". Could we say it's two layers of a single network? Is there actually a technical difference that makes claiming "two networks" more accurate/clear?

Reply:
From the EL/CL bullet points below, it seems like we refer to "the network", which further confuses me. Seems like we could stick with Ethereum being a single network but LMK what you think @taxmeifyoucan.

Reply:
Agree, neither can function on its own after The Merge

Reply:
Agreed, it's more confusing than I realized. I used the word `networks` because they both have own p2p network and I wanted to emphasize that Beacon Chain  is not a new thing and the network has been running for a long time. 

I would rephrase it as layers - `Post-Merge Ethereum consists of two parts: the execution layer and the consensus layer. Both layers are run by different client software.`